app/search_builders/hyrax/admin_set_search_builder.rb

```diff
@@ -8,6 +8,7 @@ class AdminSetSearchBuilder < ::SearchBuilder
     # @param [Symbol] access one of :edit, :read, or :deposit
     def initialize(context, access)
       @access = access
+      @rows = 100
```

Comment:
I do not think we should set `@rows` at all if we can avoid it. We can do `AdminSetSearchBuilder.new.rows(100)` instead.  That's how you're supposed to use that API.